### DISCUSSION TITLE
CoApp Installation Fails on Mac M1

### DISCUSSION THREAD

**Discussion by jazhang22** [aclap-dev/video-downloadhelper#815] [Help and Q&A]
### Your question

Hi! I have been trying to download the CoApp for Mac (with an M1 chip) but every time I install I get this error! It doesn't seem to be on the troubleshooting page. Do you have any idea what might be going wrong? Thanks in advance!

<img width="622" alt="Screenshot 2024-06-01 at 3 10 39 PM" src="https://github.com/aclap-dev/video-downloadhelper/assets/118634874/798a694e-2c22-447b-b19c-187c59750e2e">

### Addon & CoApp version and details

version: 9.0.2.3
target: google
channel: stable
lang: en-US
coapp: {"error":"Specified native messaging host not found.","found":false}
license: {"unset":true}
platform: arm64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36
{
    "used_sidebar_button": true,
    "used_popup_button": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": false,
    "open_count_store": 8
}

**Comment by cj-zhang** [comment]
Also facing this issue, would appreciate any insight here.

**Comment by paulrouget** [comment]
Hmm, that's surprising. Did you already have it installed and tried to re-install it?

Here is another way of installing it:

- Download this DMG: https://github.com/aclap-dev/vdhcoapp/releases/download/v2.0.19/vdhcoapp-mac-arm64.dmg
- Move the app in the Applications folder
- Open the App once (it should not do anything, but it's necessary to make sure the app registers itself properly)

**Reply by jazhang22** [reply]
AMAZING! Thank you so much, this worked. I had the Chrome Extension/program installed and it worked previously. But when I recently tried to use it, it said "Companion application required" (screenshot below for reference) which took me the otherwise regular download. I followed your instructions above to re-install and it's all working again. 

<img width="329" alt="Screenshot 2024-06-05 at 9 38 30 AM" src="https://github.com/aclap-dev/video-downloadhelper/assets/118634874/5fcb001e-d4d9-4362-ab4f-dcf46750a10d">

**Comment by paulrouget** [comment]
Which version of MacOS are you using?

**Reply by jazhang22** [reply]
I'm using macOS 13.0 (22A380). Apple M1 2020. Thanks again for your attention to this!
